packages/ui-components/src/icon/labicon.tsx

```diff
@@ -458,6 +458,7 @@ export class LabIcon implements LabIcon.ILabIcon, VirtualElement.IRenderer {
           label,
           title,
           tag = 'div',
+          slot,
```

Comment:
To apply the `slot` attribute in case `tag` is `null` (this use case), we need to change the logic below with the following patch

```patch
@@ -487,10 +487,21 @@ export class LabIcon implements LabIcon.ILabIcon, VirtualElement.IRenderer {
           // bail if failing silently
           return <></>;
         }
+        const svgProps = { ...this.svgReactAttrs };
+        if (!tag) {
+          Object.assign(svgProps, {
+            className:
+              className || styleProps
+                ? classes(className, LabIconStyle.styleClass(styleProps))
+                : undefined,
+            title: title,
+            slot: slot
+          });
+        }
 
         const svgComponent = (
           <svg
-            {...this.svgReactAttrs}
+            {...svgProps}
             dangerouslySetInnerHTML={{ __html: this.svgInnerHTML }}
             ref={ref}
           />
```